src/content/en/tools/workbox/reference-docs/v3.0.0-alpha.1/workbox.expiration.html

```diff
@@ -27,7 +27,7 @@ <h3 class="class-list"><a href="workbox.expiration.CacheExpiration.html">CacheEx
               <h3 class="class-list"><a href="workbox.expiration.CacheExpirationPlugin.html">CacheExpirationPlugin</a></h3>
               <p>This plugin can be used in the Workbox API's to regularly enforce a limit on the age and / or the number of cached requests.</p>
               <p>Whenever a cached request is used or updated, this plugin will look at the used Cache and remove any old or extra requests.</p>
-              <p>When using <code>maxAgeSeconds</code>, requests may be used <em>once</em> after expiring because the expiration clean up will not have occured until <em>after</em> the cached request has been used. If the request has a &quot;Date&quot; header,
+              <p>When using <code>maxAgeSeconds</code>, requests may be used <em>once</em> after expiring because the expiration clean up will not have occurred until <em>after</em> the cached request has been used. If the request has a &quot;Date&quot; header,
```

Comment:
@gauntface - are these coming from generated docs? If so, can you make this fix in the workbox docs too please?

Reply:
This *was* but we are on beta now, this is the first / second alpha, so it should be fixed.